docs/standard/garbage-collection/implementing-dispose.md

```diff
@@ -130,20 +133,6 @@ Here's the general pattern for implementing the dispose pattern for a derived cl
 :::code language="csharp" source="../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.idisposable/cs/derived2.cs":::
 :::code language="vb" source="../../../samples/snippets/visualbasic/VS_Snippets_CLR_System/system.idisposable/vb/derived2.vb":::
 
-## Implement the dispose pattern with safe handles
-
-The following example illustrates the dispose pattern for a base class, `DisposableStreamResource`, that uses a safe handle to encapsulate unmanaged resources. It defines a `DisposableStreamResource` class that uses a <xref:Microsoft.Win32.SafeHandles.SafeFileHandle> to wrap a <xref:System.IO.Stream> object that represents an open file. The class also includes a single property, `Size`, that returns the total number of bytes in the file stream.
```

Comment:
What was the reasoning for deleting the safe handle examples?

Reply:
As it is explained in bullet #8 in [Summary](https://github.com/dotnet/docs/pull/26192#issue-1000281594).

>Chapters Implement the dispose pattern with safe handles and Implement the dispose pattern for a derived class with safe handles removed since both states nothing unsaid, see. Implement the dispose pattern|1st example and Implement the dispose pattern for a derived class|1st example, about dispose pattern. Moreover example show excessive information about using safe handles, extern API, … If these should be kept, I would introduce them into some Additional Examples chapter (or to some place more related to SafeHandles).